l10n_es_aeat_mod190/models/account_move.py

```diff
@@ -39,14 +39,17 @@ class AccountMove(models.Model):
     )
 
     def add_keys(self, set_fiscal_position=True):
-        FiscalPosition = self.env["account.fiscal.position"]
         for invoice in self:
             if set_fiscal_position:
-                delivery_partner_id = invoice._get_invoice_delivery_partner_id()
-                invoice.fiscal_position_id = FiscalPosition.with_company(
-                    invoice.company_id.id
-                ).get_fiscal_position(
-                    invoice.partner_id.id, delivery_id=delivery_partner_id
+                delivery_partner_id = self.env["res.partner"].browse(
```

Comment:
The variable name should be `delivery_partner`, as it holds a recordset, not an ID.